scenariolib/visit.go

```diff
@@ -408,9 +408,10 @@ func (v *Visit) FindDocumentRankByTitle(toFind string) int {
 // WaitBetweenActions Wait a random or constant number of seconds between user actions
 func WaitBetweenActions(timeToWait int, isConstant bool) {
 	if isConstant {
-		time.Sleep(time.Duration(timeToWait) * time.Second)
+		time.Sleep((time.Duration(timeToWait*1000) + 500) * time.Second)
```

Comment:
It would be weird to input `1000` and have a `500`. Don't you want to do a simple `Math.min` or the GO equivalent?

Plus: timeToWait is now in milliseconds => old configurations are broken
And you still multiply it by seconds, so a timeToWait initially set to 1 second is now 1500 seconds :)